tests/unit/docker/test_hubapi.py

```diff
@@ -57,4 +61,46 @@ def test_fetch_access_token(mocker):
     token_string = json.dumps({'access_token':'dummy_token'})
     token_json = json.loads(token_string)
     mocker.return_value = token_json
-    _fetch_access_token(logger=getLogger())
\ No newline at end of file
+    _fetch_access_token(logger=getLogger())
+
+
+@responses.activate
+def test_docker_auth_success(mocker):
+    token_string = json.dumps({'hubapi': {'url': 'dummy_url', 'docker_auth': 'dummy_auth'}})
```

Comment:
You can get rid of the redundant code in `test_docker_auth_success` and `test_docker_auth_failure` by using fixtures.

Reply:
Done